themes/api.jquery.com/index.php

```diff
@@ -2,6 +2,14 @@
 <div class="content-right listing twelve columns">
 	<div id="content">
 		<h1 class="page-title">jQuery API</h1>
+		<p>jQuery is a fast, small, and feature-rich JavaScript library. It makes things like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API that works across a multitude of browsers. If you're new to jQuery, we recommend to check out the <a href="http://learn.jquery.com/">jQuery Learning Center</a>.</p>
+		<p>If you're wanting to update to a newer version of jQuery, you might be interested in reading the release notes, published on <a href="http://blog.jquery.com/">our blog</a>. If you're coming from a version prior 1.9, it's definitely worth it to check out the <a href="http://jquery.com/upgrade-guide/1.9/">1.9 Upgrade Guide</a> as well.</p>
```

Comment:
If you're updating to a newer version of jQuery, be sure to read the release notes published on <a href="http://blog.jquery.com/">our blog</a>.  If you're coming from a version prior to 1.9, check out…